src/audit-license-headers.js

```diff
@@ -44,11 +46,27 @@ var COMMON_RAT_EXCLUDES = [
     '*.xcworkspacedata',
     '*.xccheckout',
     '*.xcscheme',
-    ];
+];
+
+var RAT_IGNORE_PATH          = '.ratignore';
+var RATIGNORE_COMMENT_PREFIX = '#';
+
+var RAT_NAME = 'apache-rat-0.10';
+var RAT_URL  = 'https://dist.apache.org/repos/dist/release/creadur/apache-rat-0.10/apache-rat-0.10-bin.tar.gz';
+
+function startsWith(string, prefix) {
+    return string.lastIndexOf(prefix, 0) === 0;
```

Comment:
using lastIndexOf will fail with this won't it?
startsWith("boomboom","boom"); // false?
string.indexOf(prefix) === 0 is not only correct, but easier to understand too